### ISSUE TITLE
fill() from texture to display doesnt work

### ISSUE THREAD

**Issue by XLPhere** [glium/glium#1748]
Hey!
I'm trying to use a texture to buffer the result before I display it on a window.
So i used a fill() to copy the buffer from the texture to the window.
But fill() doesn't seem to work for me here though, so i just get a flickering image of the VRAM.
![image](https://user-images.githubusercontent.com/13018470/60620616-f6787980-9ddb-11e9-8fb8-d622d0905914.png)
Everything i do directly on the canvas works fine.
The clear_color() on the window-canvas and any other direct drawing on there works, though it doesn't receive any change from the fill() where i try to paste the buffer.

My system runs on a Vega64 with the amdgpu-kernel-driver (for kernel 5.1.10-1-default) and X11 so it might also be a driver issue... other platform test may help here...

Or am i just using it wrong? - I'm sorry if i did, since I'm actually pretty new to OpenGL in general..

To summarize what i do here: I create a window and prepare everything, then create a buffer for every frame, draw on the buffer and then try to fill the buffer on the window (which i think fails)

Here is my code for now: https://paste.rs/UBN.rs

Thanks for your help!
